src-docs/src/views/collapsible_nav/collapsible_nav_all.tsx

```diff
@@ -148,6 +151,11 @@ const CollapsibleNavAll = () => {
         </EuiHeaderSectionItemButton>
       }
       onClose={() => setNavIsOpen(false)}
+      css={css`
+        @media (max-height: 15em) {
+          ${logicalCSSWithFallback('overflow-y', 'auto')}
+        }
+      `} // Accessibility - Add scroll to nav on very small screens
```

Comment:
minor nit - let's move the comment above the css tag for readability
```suggestion
      // Accessibility - Add scroll to nav on very small screens
      css={css`
        @media (max-height: 15em) {
          ${logicalCSSWithFallback('overflow-y', 'auto')}
        }
      `}
```